_sass/blocks/explore/_filters.scss

```diff
@@ -209,6 +206,10 @@
     &[aria-expanded=false] {
       display: block;
     }
+
+    @include respond-to(medium-up) {
```

Comment:
Expected item on line 210 to appear before line 202. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets